### PR TITLE
added missing ingress rules

### PR DESCRIPTION
**Pull request by Loki-Afro**
# Description

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: ansible/roles/ingress/templates/ingress.yml.j2 [hunk 1]
```diff
@@ -207,6 +211,22 @@ spec:
...
+          serviceName: api-svc
+          servicePort: {{ PORT_SERVER }}
+      - path: /api/v1/systems
+        backend:
+          serviceName: api-svc
+          servicePort: {{ PORT_SERVER }}
+      - path: /api/v1/ldap-config
+        backend:
+          serviceName: api-svc
+          servicePort: {{ PORT_SERVER }}
```
Comment (mamutmk5) [inline ansible/roles/ingress/templates/ingress.yml.j2:229]: Rejected, Route is not secure for public.
The get methode is insecure

https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/src/services/ldap-config/index.js
https://github.com/hpi-schul-cloud/schulcloud-server/blob/b1e528dd5c5662f1f5c9ea14bc50d84cf2a7bf38/src/services/ldap-config/service.js#L26-L34
https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/src/services/ldap-config/hooks/index.js

File: ansible/roles/ingress/templates/ingress.yml.j2 [hunk 2]
```diff
@@ -172,6 +172,10 @@ spec:
         backend:
           serviceName: api-svc
           servicePort: {{ PORT_SERVER }}
+      - path: /api/v1/fileStorage/total
+        backend:
+          serviceName: api-svc
+          servicePort: {{ PORT_SERVER }}
```
Comment (mamutmk5) [inline ansible/roles/ingress/templates/ingress.yml.j2:178]: Rejected, the implementation says there is an securety issue

https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/src/services/fileStorage/proxy-service.js#L707-L721

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
Rejected by potential security issues.
Please add an proof that the Server Routes are secure to open.

File: ansible/roles/ingress/templates/ingress.yml.j2 (hunk 2 above)
Comment (Loki-Afro) [reply ansible/roles/ingress/templates/ingress.yml.j2:178]: the implementation is not existent, always returns 0

File: ansible/roles/ingress/templates/ingress.yml.j2 [hunk 3]
```diff
@@ -207,6 +211,22 @@ spec:
         backend:
           serviceName: api-svc
           servicePort: {{ PORT_SERVER }}
+      - path: /api/v1/years/
+        backend:
+          serviceName: api-svc
+          servicePort: {{ PORT_SERVER }}
```
Comment (mamutmk5) [inline ansible/roles/ingress/templates/ingress.yml.j2:217]: If this Route is open it's posible for persone with Permission "SCHOOL_EDIT" to generate an query for an update or patch of an school year.
https://github.com/hpi-schul-cloud/schulcloud-server/blob/3d6ee4e6ca30c1df7be5fd101af78cb3c81202ba/src/services/school/index.js#L65-L78
https://github.com/hpi-schul-cloud/schulcloud-server/blob/3d6ee4e6ca30c1df7be5fd101af78cb3c81202ba/src/services/school/index.js#L65-L78

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: ansible/roles/ingress/templates/ingress.yml.j2 (hunk 2 above)
Comment (mamutmk5) [reply ansible/roles/ingress/templates/ingress.yml.j2:178]: Okay, fine for me at the moment.
But maybe the function shoud be fixed at futrue.

File: ansible/roles/ingress/templates/ingress.yml.j2 (hunk 1 above)
Comment (mamutmk5) [reply ansible/roles/ingress/templates/ingress.yml.j2:229]: Route Fix, is open right now

File: ansible/roles/ingress/templates/ingress.yml.j2 (hunk 3 above)
Comment (mamutmk5) [reply ansible/roles/ingress/templates/ingress.yml.j2:217]: hpi-schul-cloud/schulcloud-server#2958

**Review by mamutmk5 — DISMISSED** [review]

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**